aten/src/ATen/native/native_functions.yaml

```diff
@@ -8284,6 +8284,11 @@
 #
 # See linalg_det as an example.
 
+- func: linalg_cholesky(Tensor self) -> Tensor
```

Comment:
What's the plan for the out variant of this function?

Reply:
I added it now.